srcpkgs/Orthanc/files/Configuration.json

```diff
@@ -0,0 +1,915 @@
+{
```

Comment:
adding to distfiles will prevent this from going out of sync

Reply:
It's already distributed on distfile, I just changed some default settings (specifically to match $ORTHANCSTORAGE on service). I think I'll run a sed on it instead.

Reply:
a patch would probably be better

Reply:
@classabbyamp I done it